@ecomplus/storefront-components/src/js/ShippingCalculator.js

```diff
@@ -154,7 +154,7 @@ export default {
           if (!validated || error) {
             return
           }
-          if (this.skipAppIds && this.skipAppIds.includes(appResult.app_id)) {
+          if (this.skipAppIds && this.skipAppIds.includes(appResult.app_id) && (shippingResult.length !== 1)) {
```

Comment:
```suggestion
          if (
            this.skipAppIds &&
            this.skipAppIds.includes(appResult.app_id) &&
            shippingResult.filter(({ app_id: appId }) => !this.skipAppIds.includes(appId)).length
          ) {
```

Se eu entendi bem é isso aí que c quer, não?
Quer dizer, não tá relacionado a ter só um resultado, tá relacionado a ter pelo menos um resultado sem ser skipado...

Reply:
Isso, ficou melhor. No caso, se tiver um, nao deve ser skippado, somente se tiver mais de um, nesse caso ai ficou melhor